packages/nocodb/src/models/User.ts

```
@@ -367,4 +386,27 @@ export default class User implements UserType {
+
+  public static async signUserImage(
+    users: Partial<UserType> | Partial<UserType>[],
+  ) {
+    if (!users) return;
+
+    const promises = [];
+    
+    try {
+      for (const user of Array.isArray(users) ? users : [users]) {
+        const meta = parseMetaProp(user);
+        if (meta && meta.icon && meta.iconType === IconType.IMAGE) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)